src/data/travel/airports.rs

```diff
@@ -0,0 +1,408 @@
+use crate::utils::seeder;
+use wasm_bindgen::prelude::*;
+
+// TODO need help with what to do with flag bytes array
+// Maybe some categories? asia, north america.. etc
+// Could add more languages
+
+#[wasm_bindgen]
+pub fn random_us_airport() -> String {
+	US_AIRPORT_NAMES[seeder::gen_range(0..US_AIRPORT_NAMES_LEN)].to_string()
+}
+
+#[wasm_bindgen]
+pub fn random_us_airport_iata() -> String {
+	US_AIRPORT_CODES[seeder::gen_range(0..US_AIRPORT_CODES_LEN)].to_string()
+}
+
+#[wasm_bindgen]
+pub fn random_eu_airport() -> String {
+	EU_AIRPORT_NAMES[seeder::gen_range(0..EU_AIRPORT_NAMES_LEN)].to_string()
+}
+
+#[wasm_bindgen]
+pub fn random_eu_airport_iata() -> String {
+	EU_AIRPORT_CODES[seeder::gen_range(0..EU_AIRPORT_CODES_LEN)].to_string()
+}
```

Comment:
I say remove the `random_` prefix in the function names. All the other generator functions are random by default already this seems like a redundant naming scheme.